wren-ui/src/components/pages/setup/utils.tsx

```diff
@@ -15,6 +15,7 @@ import SQLServerProperties from './dataSources/SQLServerProperties';
 import ClickHouseProperties from './dataSources/ClickHouseProperties';
 import { SampleDatasetName } from '@/apollo/client/graphql/__types__';
 import { ERROR_CODES } from '@/utils/errorHandler';
+import { TrinoProperties } from './dataSources/TrinoProperties';
```

Comment:
Following our coding style, I suggest: 
```suggestion
import TrinoProperties from './dataSources/TrinoProperties';
```
```diff
// wren-ui/src/components/pages/setup/dataSources/TrinoProperties.tsx

- export function TrinoProperties({ mode }: Props) {
+ export default function TrinoProperties({ mode }: Props) {
```